### PR TITLE
e2e tests for OCI commands

### LINKED ISSUE #3658: Migrate OCI tests from #3486
It appears some e2e testing for the OCI runtime were added to #3486 under `e2e/scripts/oci/`. I am not aware of equivalent tests within the current framework(@cclerget correct me if I'm wrong), so equivalent tests should be made.

### PR DESCRIPTION
**Pull request by cclerget**
**Description of the Pull Request (PR):**

Fixes #3658

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: e2e/oci/oci.go [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+	defer umountFn()
+
+	e2e.RunSingularity(
+		t,
+		"OciAttachCreate",
+		e2e.WithPrivileges(true),
+		e2e.WithCommand("oci create"),
+		e2e.WithArgs("-b", bundleDir, containerID),
+		// this is required otherwise oci create hangs, this is
+		// due to command Wait call waiting that executed command
```
Comment (gvallee) [inline e2e/oci/oci.go]:
```suggestion
		// due to command Wait call that waits for the execution of a command
```

File: e2e/oci/oci.go [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+		// close standard file descriptor, but OCI create keep
```
Comment (gvallee) [inline e2e/oci/oci.go]:
```suggestion
		//  that closes standard file descriptors, but OCI create keeps
```

File: e2e/oci/oci.go [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+	// umount bundle
+	defer umountFn()
+
+	e2e.RunSingularity(
+		t,
+		"OciBasicCreate",
+		e2e.WithPrivileges(true),
+		e2e.WithCommand("oci create"),
+		e2e.WithArgs("-b", bundleDir, containerID),
+		// this is required otherwise oci create hangs, this is
```
Comment (gvallee) [inline e2e/oci/oci.go:220]: Maybe update the comment like above

**Review by gvallee — CHANGES_REQUESTED** [review]
Only minor comments...

File: e2e/oci/oci.go (hunk 3 above)
Comment (cclerget) [reply e2e/oci/oci.go:220]: Will do, thanks !

**Review by gvallee — APPROVED** [review]
LGTM

**Pull request merged**